sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -641,21 +649,28 @@ else if (mUnknownSourcesDialog != null) {
                 return;
             }
 
-            /* Check if we have previous stored the update token. */
+            /*
+             * Check if we have previously stored the redirection parameters,
+             * note that distribution group is missing in previous SDK releases.
```

Comment:
Let's do that. This will be released by me or after 3 weeks and we can easily forget about it.